src/metabase/models/recent_views.clj

```diff
@@ -69,16 +72,30 @@
                 ;;       distinguish between them.
    :dashboard :table :collection])
 
-(defn- ids-to-prune-for-user+model [user-id model]
+(defn- moi->type
```

Comment:
I am purposely using a synthetic name (as opposed to a natural name) to refer to `models of interest` (here `moi`). We take them in as `model` on the activity api 🫠, and there's a mapping from `moi`s to models, but they aren't the same so I don't think we should call them the same thing.

Reply:
I can't help but read `moi` as a French word and pronounce it in my head accordingly

maybe `recent-views-model`/`rv-model` or something like that? to make it clear that it's the "model" field of recent views, rather than the normal toucan model identifier.

Reply:
(or the model card type).

Yea, I'll switch `moi` to another synthetic name.